### DISCUSSION TITLE
Unified dashboard across clouds

### DISCUSSION THREAD

**Discussion by mlabouardy** [tailwarden/komiser#199] [Ideas]
Ability to build a custom dashboard with different components/widgets across different cloud providers (AWS, GCP, Azure, etc)

What do you guys think?

**Comment by rkujawa** [comment]
This would definitely be useful. Currently web interface is not customisable at all. This means that even if I use a single cloud provider, there's a bunch of stuff I don't really use (and it just clutters the web interface). So I'd say, if komiser is going to differentiate from the competition, ability to customise the UI is a must.
